recipes/ocgis/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "1.3.0" %}
+
+package:
+    name: ocgis
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/NCPP/ocgis.git
+    #git_tag: v{{ version }}
+    git_tag: i415-conda-forge
```

Comment:
I re-started the CIs. If that works do you plan on a new release or do you want to make the conda package using this branch for a while? (That is fine if you have reasons to do so.)